tasks/configure.yml

```diff
@@ -8,17 +8,13 @@
     mode: 0444
     owner: root
     src: daemon.json.j2
-  register: docker_config_file
-
-- name: Ensure Docker service is restarted with configuration changes
-  become: yes
-  service:
-    name: docker
-    state: restarted
-  when: docker_config_file.changed
+  notify:
```

Comment:
I think we need to keep the task here as opposed to the handler, if I remember correctly Docker needs to be restarted immediately if the thinpool configuration changes.

Reply:
I'll flush the handlers instead.

Reply:
I don't think that's a good solution, you will flush all handlers everywhere with that call, I would stick to the targeted method that we have in place already.